changelogs/fragments/344.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  postgres_exporter - Filter out idle-in-transaction sessions from general max query runtime metrics.
```

```suggestion
  - postgres_exporter - Filter out idle-in-transaction sessions from general max query runtime metrics.
```
antsibull-changelog expects a list